### ISSUE TITLE
distutils has been deprecated in Python 3.10

### ISSUE THREAD

**Issue by tirkarthi** [makinacorpus/django-leaflet#346]
https://www.python.org/dev/peps/pep-0632/#migration-advice

https://github.com/makinacorpus/django-leaflet/blob/f2d421632a065cdcac5fd3b3aac815f8e84df412/leaflet/__init__.py#L2

**Comment by claudep** [comment]
Looks like a "simple" `if django.VERSION < (3, 2):` would do the job and avoid requiring LooseVersion.

**Comment by Gagaro** [comment]
Thanks for the report, it's fixed :+1: .

**Comment by rodolvbg-divineslns** [comment]
Hi, @Gagaro, it seems that you did not release a new version after this change. I still get this warning. Could you please release a new one.
```
venv3.10/lib/python3.10/site-packages/leaflet/__init__.py:187: 10 warnings
  /home/charlie/PycharmProjects/maroon-bells/venv3.10/lib/python3.10/site-packages/leaflet/__init__.py:187: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if LooseVersion(django.__version__) < LooseVersion('3.2'):
```

**Comment by Gagaro** [comment]
Released in 0.28.3
